manimlib/mobject/svg/code_mobject.py

```diff
@@ -1,52 +1,118 @@
+        Name of the code file to display.
+    tab_width : :class:`int`, optional (default is 3).
+        Number of space characters for a tab character.
+    line_spacing : :class:`float`, optional (default is 0.3, which means 30% of font size)
+        Amount of space between lines in respect of font size.
+    scale_factor : class:`float`, optional (default is 0.5)
+        It is a number which scales displayed code.
+    font : :class:`str`, optional (default is 'Monospac821 BT')
+         Name of text font.
+    stroke_width : class:`float`, optional (default is 0)
+        Stroke width for text. 0 is recommended.
+    margin: class :`float`, optional (default is 0.3)
+        Inner margin of text from background.
+    indentation_chars : :class:`str`, optional (default is "    ")
+        Indentation chars refers to the spaces/tabs at the beginning of given code line.
+    background : :class:`str`, optional (default is "rectangle")
+        It defines background type. Currently supports only "rectangle" and "window".
+    background_stroke_width : class:`float`, optional (default is 1)
+        It defines stroke width of background.
+    background_stroke_color : class:`str`, optional (default is WHITE)
+        It defines stroke color for background.
+    corner_radius : :class:`float`, optional (default is 0.2)
+        It defines corner radius for background.
+    insert_line_no : :class:`bool`, optional (default is True)
+        It defines whether insert line numbers in displayed code.
+    line_no_from : :class:`int`, optional (default is 1)
+        It defines starting number for counting lines.
+    line_no_buff : :class:`float`, optional (default is 0.4)
+        It defines space between line numbers and displayed code.
+    style : :class:`str`, optional (default is 'vim')
+        It defines style type of displayed code. You can see names of styles from Code.styles_list.
+    language : :class:`str`, optional (default is None, Which mean it will automatically detects the language)
+        It defines the language name of given code.
+        For Knowing more available options visit https://pygments.org/docs/lexers/  for 'aliases or short names'
+    generate_html_file : :class:`bool`, optional (default is False)
+        It defines whether to generate code highlighted html to folder assets/codes/generated_html_files.
+    Attributes (It have all Parameters as Attributes and some extra as following)
+    ----------
+    background_mobject : :class:`~.VGroup`
+        To display background according to background type specified by background in Parameters.
+        VGroup with SurroundingRectangle() if background == "rectangle"
+        VGroup with RoundedRectangle() and Dot() for three buttons if background == "window"
+    line_numbers : :class:`~.Paragraph`
+        To display line numbers of displayed code if insert_line_no == True.
+    code : :class:`~.Paragraph`
+        To display highlighted code.

```

Comment:
what do you mean here by these "to display"s?

```suggestion
        The background displayed according to the background type specified by `background` in Parameters.
        This is a VGroup with :class:`~.SurroundingRectangle` if `background == "rectangle"`, and a VGroup with :class:`~.RoundedRectangle` and :class:`~.Dot` (three Dots for three buttons)) if `background == "window"`.
    line_numbers : :class:`~.Paragraph`
        The line numbers of displayed code if `insert_line_no == True`.
    code : :class:`~.Paragraph`
        The displayed highlighted code.
```